tests/test_sessions.py

```diff
@@ -0,0 +1,75 @@
+import os
+import tempfile
+import pytest
+import json
+from flasksrc import create_app, db
+from flask_jwt_extended import create_access_token
+
+@pytest.fixture
+def client():
```

Comment:
please also test the `put` method of `ToyNetSessionById` and then `get` again to ensure updates persist and are retrievable.

Reply:
need to add a schema for ToyNetSessionByIdPutReq